### DISCUSSION TITLE
Multi-server (Bungee) support

### DISCUSSION THREAD

**Discussion by EvilPandaMan** [EssentialsX/Essentials#4116] [Ideas and Feature Suggestions] [answered]
I have searched far and wide on the internet for a decent multi-server chat support but there is nothing that compares with the quality of Essentials

Could you please add the multi-server support. That would be great. I would even pay for that.

Thanks

EPM

**Comment by pop4959** [comment, marked as answer]
Essentials isn't going to have bungee support ever in the foreseeable future. It's not planned. Essentials is meant to be installed on each individual server, not the proxy.
